### DISCUSSION TITLE
How to hide title bar

### DISCUSSION THREAD

**Discussion by VladimirCores** [linuxdeepin/developer-center#2987] [Q&A | 问答板块]
Good day deepin team,
In Chrome I've found the option to hide the "title bar" of the app, it takes too much space on the screen and I'm wondering how can it be done for other system applications? 
![S](https://user-images.githubusercontent.com/442111/93643962-52d7b100-fa01-11ea-994b-b08e2b5a2b1d.png)

Peace,
VC

**Comment by BLumia** [comment]
DDE/deepin applications are using Client-Side Decoration which means they don't have a title-bar from the window manager, so actually there isn't a title bar for hiding ;p
